core/server/common/src/main/java/alluxio/master/BackupManager.java

```diff
@@ -321,4 +339,23 @@ private void safeWaitTasks(Set<Future<?>> activeTasks, CompletionService<?> comp
       }
     }
   }
+
+  /**
+   * Class that contains metrics for BackupManager.
+   * This class is public because the counter names are referenced in
+   * {@link alluxio.web.WebInterfaceAbstractMetricsServlet}.
+   */
+  public static final class Metrics {
+    private static final Timer BACKUP_ENTRIES_PROCESS_TIME
+        = MetricsSystem.timer(MasterMetrics.BACKUP_ENTRIES_PROCESS_TIME);
+    private static final Timer BACKUP_RESTORE_PROCESS_TIME
+        = MetricsSystem.timer(MasterMetrics.BACKUP_RESTORE_PROCESS_TIME);
+    private static final Counter LAST_BACKUP_ENTRIES_COUNT
+        = MetricsSystem.counter(MasterMetrics.LAST_BACKUP_ENTRIES_COUNT);
+    private static final Counter LAST_BACKUP_RESTORE_COUNT
+        = MetricsSystem.counter(MasterMetrics.LAST_BACKUP_RESTORE_COUNT);
```

Comment:
Do the timers keep track of multiple cycles while the counters only keep track of the last one? If so should these be consistent?

Reply:
Unfortunately yes. Having the entries metric report the entry count of the latest backup gives a good idea of the existing backup size. The rolling timer is useful in understanding how the backup cycle has been changing over time in duration. I'm open to suggestions on other metrics that we could collect for this, but I'm comfortable with these two metrics not being consistent because they serve different functions.